app/views/estimate_flow/household_flow/_partner_assets.html.slim

```diff
@@ -8,9 +8,6 @@
            show_smod_content: false,
            hints: t("estimate_flow.partner_assets.hints")
 
-  - translation_text = @check.controlled? ? "controlled" : "certificated"
 = render "shared/question_sidebar", \
```

Comment:
Just a question, not a blocker. Why would it not be like this following the pattern in similar partials? 

```
= render "shared/question_sidebar",
    level_of_help: @check.level_of_help,
    links: partner_assets_links(@check)

```

Reply:
I'm not sure I understand the question - what we're doing _is_ that, isn't it?

Reply:
The is an errand '\' in there.